dask_xgboost/core.py

```diff
@@ -95,20 +94,24 @@ def train_part(env, param, list_of_parts, dmatrix_kwargs=None, **kwargs):
     if labels[0] is not None:
         data = concat(data)                 # Concatenate many parts into one
         labels = concat(labels)
-        #dmatrix_kwargs["feature_names"] = getattr(data, 'columns', None)
         if dmatrix_kwargs is None:
             dmatrix_kwargs = {}
         dtrain = xgb.DMatrix(data, labels, **dmatrix_kwargs)
 
     elif labels[0] is None and isinstance(data[0], xgb.DMatrix):
         dtrain = data[0]
-        #dmatrix_kwargs["feature_names"] = getattr(data, 'columns', None)
         if dmatrix_kwargs is None:
             dmatrix_kwargs = {}
 
+    elif labels[0] is None and isinstance(data[0], gd):
```

Comment:
Should it be:
elif labels[0] is None and isinstance(data[0], gd.DataFrame):

?

Reply:
Corrected.